### ISSUE TITLE
ERROR: Exception: stream inflate: inflate: data: incorrect data check

### ISSUE THREAD

**Issue by akorobkin** [qpdf/qpdf#106]
pdftopdf built with qpdf 5.1.2 is stumbling upon this error when processing this and similar PDFs:
```
WARNING: ./file.pdf (file position 101510): error decoding stream
data for object 39 0: stream inflate: inflate: data: incorrect data check
ERROR: Exception: stream inflate: inflate: data: incorrect data check
```

https://shop.groveko.nl/images/products/7010209/VIB%20Suma%20Inox%20Classic%20D7%202016-03-31.pdf

GhostScript treats the error as warning and proceeds normally, Adobe Pro also proceeds normally on processing the file. 

Is it possible to make qpdf recover from this error and treat it as a warning
[VIB Suma Inox Classic D7 2016-03-31.pdf](https://github.com/qpdf/qpdf/files/825077/VIB.Suma.Inox.Classic.D7.2016-03-31.pdf)
?

**Comment by akorobkin** [comment]
Similar file with the same errors: http://fmo.unl.edu/sds/shineline.pdf

**Comment by jberkenbilt** [comment]
Yes. this is a known issue. Making qpdf recover properly from damaged streams and other damaged objects is the highest priority item on my to-do list. I have had very limited time over the last three years to work on qpdf, but I'm hoping that will change soon. There are several open issues that are all essentially this same thing.

Just for historical context, qpdf started its life originally as a PDF checker, so it was very strict. My intention is to have it be looser in recovering from bad files by default but to have an option for being strict so that people who are creating PDF files can use it to verify correct PDF, at least syntactically correct PDF. In the real world, people seem to test their PDFs by seeing of Adobe Reader can handle them, and Adobe Reader is very tolerant, as any good reader should be, of erroneous files.

**Comment by jberkenbilt** [comment]
This will be fixed for 7.0.0. My working branch, to be merged momentarily, is able to recover this file.
